src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -7468,6 +7468,7 @@
           - LaundryWasherControls
           - DishwasherAlarm
           - SampleMEI
+            ServiceArea
```

Comment:
Seemed like the best place to put the service cluster for now.  Not sure how much of this is getting used and how much it's a 'future'.

Reply:
I'm not sure what this thing does. Usually I do not touch anything to do with Darwin. It's quite nuanced so I get Boris from Apple to take care of things.

Reply:
Petru pointed out that to be consistent with the other clusters, there should be a "-".

Given your comment you probably should just revert this file to unchanged.

Reply:
I'm reverting this change. I think this is something that is maintained and used by apple engineers for their systems so they know what needs to be done here.